I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 461, in script
    "(por que soy asi de idiota, no puedo creerlo)"
Exception: Image 'bg blackbg' does not accept attributes 'depressed'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 461, in script
    "(por que soy asi de idiota, no puedo creerlo)"
  File "C:\Users\blood\Pirated\renpy\renpy-8.1.3-sdk\renpy\ast.py", line 1562, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "C:\Users\blood\Pirated\renpy\renpy-8.1.3-sdk\renpy\ast.py", line 1391, in show_imspec
    renpy.config.show(name,
  File "C:\Users\blood\Pirated\renpy\renpy-8.1.3-sdk\renpy\exports.py", line 733, in show
    if not base.find_target() and renpy.config.missing_show:
  File "C:\Users\blood\Pirated\renpy\renpy-8.1.3-sdk\renpy\display\image.py", line 421, in find_target
    self.target = target._duplicate(a)
  File "C:\Users\blood\Pirated\renpy\renpy-8.1.3-sdk\renpy\display\core.py", line 499, in _duplicate
    args.extraneous()
  File "C:\Users\blood\Pirated\renpy\renpy-8.1.3-sdk\renpy\display\core.py", line 362, in extraneous
    raise Exception("Image '{}' does not accept attributes '{}'.".format(
Exception: Image 'bg blackbg' does not accept attributes 'depressed'.

Windows-10-10.0.19041 AMD64
Ren'Py 8.1.3.23091805
EmoCat 0.0.3
Thu Dec  7 14:00:16 2023
